### PR TITLE
Create plugin-valid-network

### PR DESCRIPTION
**Pull request by dimabru**
Please allow release permissions for valid-network-plugin
git repo: https://github.com/jenkinsci/valid-network-plugin
link to hosting request with permissions for user "dima": https://issues.jenkins-ci.org/browse/HOSTING-999

# Description

# Submitter checklist for adding or changing permissions

### Always

- [x] Add link to plugin/component Git repository in description above

### For a newly hosted plugin only

- [x] Add link to resolved HOSTING issue in description above

### For a new permissions file only

- [x] Make sure the file is created in `permissions/` directory
- [x] `artifactId` (pom.xml) is used for `name` (permissions YAML file).
- [x] [`groupId` / `artifactId` (pom.xml) are correctly represented in `path` (permissions YAML file)](https://github.com/jenkins-infra/repository-permissions-updater/#managing-permissions)
- [x] Check that the file is named `plugin-${artifactId}.yml` for plugins

### When adding new uploaders (this includes newly created permissions files)

- [x] [Make sure to `@`mention an existing maintainer to confirm the permissions request, if applicable](https://github.com/jenkins-infra/repository-permissions-updater/#requesting-permissions)
- [x] Use the Jenkins community (LDAP) account name in the YAML file, not the GitHub account name
- [x] Make sure to `@`mention the users being added so their GitHub account names are known if they require GitHub merge access (see below).
- [x] [All newly added users have logged in to Artifactory at least once](https://github.com/jenkins-infra/repository-permissions-updater/#requesting-permissions)

### Reviewer checklist (not for requesters!)

- [x] Check this if newly added person also needs to be given merge permission to the GitHub repo (please @ the people/person with their GitHub username in this issue as well). If needed, it can be done using an [IRC Bot command](https://jenkins.io/projects/infrastructure/ircbot/#github-repo-management)
- [x] Check that the `$pluginId Developers` team has `Admin` permissions while granting the access.
- [x] In the case of plugin adoption, ensure that the Jenkins Jira default assignee is either removed or changed to the new maintainer.
- [x] If security contacts are changed (this includes add/remove), ping the security officer (currently `@daniel-beck`) in this pull request. If an email contact is changed, wait for approval from the security officer.

There are [IRC Bot commands](https://jenkins.io/projects/infrastructure/ircbot/#issue-tracker-management) for it

### CODE REVIEW

**Comment by timja** [comment]
```
> User name not known to Artifactory: dima
```

File: permissions/plugin-valid-network.yml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+name: "valid-network"
+github: "jenkinsci/valid-network-plugin"
+paths:
+- "org/jenkins-ci/plugins/valid-network"
```
Comment (timja) [inline permissions/plugin-valid-network.yml:5]: you had the previous group id here, which you aren't using (all new plugins use io.jenkins.plugins)
```suggestion
- "io/jenkins/plugins/valid-network"
```

**Comment by dimabru** [comment]
@timja Thanks for approving this pull request.
How can I merge it?

**Comment by timja** [comment]
@dimabru I was waiting for CI after fixing the typo

**Pull request merged**